doc/tools/parse_output_variables.py

```diff
@@ -213,13 +213,13 @@ def main():
             file_contents = f.read()
 
             # first warn about commented lines
-            p = re.compile('//\s*SetupOutputVariable')
+            p = re.compile(r'//\s*SetupOutputVariable')
```

Comment:
PEP was complaining about the escapes in these, just tell Python they are raw string literals and it won't investigate the escapes.